packages/web/config/feature-flag.ts

```diff
@@ -55,6 +61,14 @@ export const PromotedLBPPoolIds: {
     ]
   : [];
 
+// The pool ids white listed that would merge their internal and external gauges
+export const MergesInternalAndExternalIncentiveGaugeList: Record<
+  string,
+  boolean
+> = {
+  "803": true,
+};
+
```

Comment:
IMO you could remove these configs

We should just include APR calculations for gauges in the allow list

This would be more config that @JeremyParish69 would maintain.